src/components/Scheduling/AdvancedScheduling.vue

```diff
@@ -102,9 +101,16 @@ const hasManyProposals = () => {
 
 const handleDisplay = (display) => {
   step.value = display
+  emitSelections()
   emits('updateDisplay', display)
 }
 
+const handleObjectSelection = (object) => {
+  selectedObject.value = object
```

Comment:
nit-picky but variable naming "object" could be confusing since "Object" is a reserved special thing in javascript.